tests/test_security_stop_report.py

```
@@ -0,0 +1,100 @@
+    from swarm.api.routes.runs_control import _write_stop_report
+    from swarm.api.routes.runs_models import StopReportInfo
+
+@pytest.mark.anyio
+async def test_write_stop_report_security():
+    """
+    Verify that _write_stop_report properly escapes user inputs to prevent
+    Markdown injection and Stored XSS.
+    """
+    # Setup
+    runs_root = Path("./test_runs_security")
+    if runs_root.exists():
```

Ruff (0.15.2)

[warning] Async functions should not use pathlib.Path methods, use trio.Path or anyio.path

(ASYNC240)